monai/__init__.py

```diff
@@ -13,9 +13,53 @@
 
 import os
 import sys
-
+import logging
+import warnings
 from ._version import get_versions
 
+
+old_showwarning = warnings.showwarning
+
+
+def custom_warning_handler(message, category, filename, lineno, file=None, line=None):
+    ignore_files = ["ignite/handlers/checkpoint", "modelopt/torch/quantization/tensor_quant"]
```

Comment:
I guess we can expect this is not needed after the packages are upgraded to fix these warnings. So please add some notes or TODO item here. Thanks!